packages/flutter/lib/animation.dart

```diff
@@ -6,9 +6,157 @@
+/// [Animation] object also stores the current [AnimationStatus]. The status
+/// indicates whether the animation is currently conceptually running from the
+/// beginning to the end or the other way around. It may also indicate that the
+/// animation is currently stopped at the beginning or the end.
+///
+/// Other objects can register listeners on an [Animation] to be informed
+/// whenever the animation value and/or the animation status changes. A [Widget]
+/// may register such a *value* listener to rebuild itself with the current
+/// animation value whenever that value changes. For example, a widget might
+/// listen to an animation to update its opacity to the animation's value every
+/// time that value changes. Likewise, registering a *status* listener may be
+/// useful to trigger another action when the current animation has ended.
+///
+/// As an example, the following video shows the changes over time in the
+/// current animation status and animation value for the opacity animation of a
+/// widget. This [Animation] is driven by an [AnimationController] (see next
+/// section). Before the animation triggers, the animation status is "dismissed"
+/// and the value is 0.0. As the value runs from 0.0 to 1.0 to fade in the
+/// widget, the status changes to "forward". When the widget is fully faded in
+/// at an animation value of 1.0 the status is "completed". When the animation
+/// triggers again to fade the widget back out, the animation status changes to
+/// "reverse" and the animation value runs back to 0.0. At that point the widget
+/// is fully faded out and the animation status switches back to "dismissed"
+/// until the animation is triggered again.
+///
+/// {@animation 420 100 https://flutter.github.io/assets-for-api-docs/assets/animation/animation_status_value.mp4}
+///
+/// Although you can't instantiate [Animation] directly (it is an abstract
+/// class) you can create one using an [AnimationController].
+///
+///
+/// ### Powering animations: AnimationController
+///
+/// An [AnimationController] is a special kind of [Animation] that advances its
+/// animation value whenever the device running the application is ready to
+/// display a new frame (typically, this rate is around 60 values per second).
+/// An [AnimationController] can be used wherever an [Animation] is expected. As
+/// the name implies, an [AnimationController] also provides control over its
+/// [Animation]: It implements methods to stop the animation at any time and to
+/// run it forward as well as in the reverse direction.
+///
+/// By default, an [AnimationController] increases its animation value linearly
+/// over the given duration from 0.0 to 1.0 when run in the forward direction.
+/// For many use cases you might want the value to be of a different type,
+/// change the range of the animation values, or change how the animation moves
+/// between values. This is achieved by wrapping the animation: An [Animatable]
```

Comment:
I'm not sure I understand this sentence with the colon. Do you wrap the animation in an Animatable?

Reply:
Yes. Rephrased.